asreview/webapp/src/PreReviewComponents/ProjectAlgorithms.js

```diff
@@ -48,7 +53,9 @@ const useStyles = makeStyles(theme => ({
 }));
 
 const defaultModels = {
-  model: "nb"
+  model: "nb",
+  query_strategy: "max",
+  feature_extraction: "tfidf",
 }
```

Comment:
This should be a GET request in the future: https://github.com/asreview/asreview/pull/316. This avoids defining the defaults twice.

Reply:
I think this relates to something @qubixes and I talked about last week. 

Is it possible to request models from a configuration file? (in the background) Similar to the option `--config-file config.ini` in the CLI. Might be convenient when we scale up to more complex configurations (hyperparameter specifications etc).

Reply:
Not completely sure that I understand you correctly. I think all models should be registered centrally. This makes it easy to pass them to the front-end. Plugins can be used to add new models.

Reply:
> This should be a GET request in the future: #316. This avoids defining the defaults twice.

Resolved.

Reply:
I like this solution. It differs from what I was thinking of, but seems to work. Let me check for some side effects before merging.

Reply:
> Not completely sure that I understand you correctly. I think all models should be registered centrally. This makes it easy to pass them to the front-end. Plugins can be used to add new models.

@J535D165 What do you mean with registred centrally? I am not talking about adding new models, but implementing additional features such as simulation and hyperparameter settings. Maybe it doesn't work that way, but I think if the models are requested through a configuration file, than later on, this file can also be used to specify settings simulations, and hyperparameters. 

I mean files like this one: 

```
[global_settings]
n_instances = 1
n_prior_included = 1
n_prior_excluded = 1
model = nb
balance_strategy = double
query_strategy = max
feature_extraction = tfidf

[balance_param]
a = 1249.910315374313
alpha = 0.7043792455091337
b = 0.9993171869006371

[feature_param]
ngram_max = 2
split_ta = 1
use_keywords = 1

[model_param]
alpha = 0.17584727645029802
 
```

Reply:
@J535D165 I think long term it might be good to work with the configuration files that @GerbrichFerdinands mentions. Because that should give better performance for the user than using the defaults for each of the individual model parts (classifier/balance/feature). Then allowing the user to upload these configuration files makes it so that advanced users can use any combination + set parameters as they want.

The list here could be a drop down list of "curated" combinations, where we can refer to the simulation study by @GerbrichFerdinands to indicate the performance:

NB+TFIDF ~> 100%
SVM+TFIDF ~> 97%

And perhaps some indication of running time as well, though I'm not sure how useful that is in this context:

NB+TFIDF ~> 2 seconds initialization, 0.1s running time.
Logistic+Doc2Vec ~> 10 min initialization, 0.1s running time.

Just some ideas. :)

Reply:
> The list here could be a drop down list of "curated" combinations, where we can refer to the simulation study by @GerbrichFerdinands to indicate the performance:
NB+TFIDF ~> 100%
SVM+TFIDF ~> 97%
And perhaps some indication of running time as well, though I'm not sure how useful that is in this context:
NB+TFIDF ~> 2 seconds initialization, 0.1s running time.
Logistic+Doc2Vec ~> 10 min initialization, 0.1s running time.

I like this one.

Levels of automation:
```
0. Use the defaults
1. Pick one of the combinations (like you propose)
2. Select with the tool made by @terrymyc. 
3. Upload a file with settings
```

We should think about the UX with @MartijnUX